proto/backend.proto

```diff
@@ -45,6 +45,20 @@ message DataQuery {
   bytes json = 5; // Arbitrary structure (model?  modelJson in v1)
 }
 
+// based on https://github.com/grafana/grafana/blob/master/pkg/models/user.go#L170
+message User {
+  int64 userId = 1;
+  int64 orgId = 2;
+  string orgName = 3;
+  string orgRole = 4;
+  string login = 5;
+  string name = 6;
+  string email = 7;
+  bool isAnonymous = 8;
+  bool isGrafanaAdmin = 9;
```

Comment:
maybe just:
* login
* name
* email
* role (viewer|editor|admin)

And maybe enterprise can send `team` as a GRPC property?

Reply:
:+1: